### PR TITLE
qs svm

### LINKED ISSUE #5: qs svm
Implement SVM Model serving using queryable state

### PR DESCRIPTION
**Pull request by mmziyad**
Implement SVM Model serving using queryable state.
fixes #5

### CODE REVIEW

**Comment by mmziyad** [comment]
@d-behi : The changes we discussed are added. Please review.

File: flink-queryable-client/src/main/java/de/tub/it4bi/modelserving/qs/SVMPredict.java [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+                out.printf("[info] Querying the model for vector '%s'\n", line);
+                try {
+                    String tokens[] = line.trim().split(" ");
+                    double prediction;
+                    double rawValue = 0;
+                    for (String s : tokens) {
+                        String pair[] = s.split(":");
+                        String id = pair[0];
+                        double val = Double.parseDouble(pair[1]);
+
```
Comment (dbehrouz) [inline flink-queryable-client/src/main/java/de/tub/it4bi/modelserving/qs/SVMPredict.java:72]: I think we have discussed this before, is it not possible to ask for multiple keys at once? to reduce the API calls?

**Review by dbehrouz — COMMENTED** [review]
I only had one concern. The code might create a bottleneck when querying the system for prediction.

File: flink-queryable-client/src/main/java/de/tub/it4bi/modelserving/qs/SVMPredict.java (hunk 1 above)
Comment (mmziyad) [reply flink-queryable-client/src/main/java/de/tub/it4bi/modelserving/qs/SVMPredict.java:72]: Do you mean is there a batch API to access queryable state in rocksdb? I think its only accessible as one query per key. This version is a simple client. I'm developing a client for experiments (issue #7 ), there we can try two things.
1. Change the parameter
query.client.network-threads: number of network (event loop) threads for the KvStateClient (0 => number of available cores)
2. Client side caching of key values

Comment (dbehrouz) [reply flink-queryable-client/src/main/java/de/tub/it4bi/modelserving/qs/SVMPredict.java:72]: 👍  then

**Pull request merged**